README.md

```diff
@@ -102,6 +122,7 @@ This also saves a list of all installed files to `install_log.txt`, which you ca
 use to uninstall ranger.
 
 
+
```

Comment:
Uhm? Let's keep in mind to remove this again at some point? Triple blank line doesn't serve any purpose afaik?

Reply:
Rightio.

Reply:
I'll fix that in a PR tomorrow.